<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StylePalette</class>
 <widget class="QDockWidget" name="StylePalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>524287</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Styles</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="styleFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>[no style file]</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="buttonNewTikzstyles">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../tikzit.qrc">
          <normaloff>:/images/document-new.svg</normaloff>:/images/document-new.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonOpenTikzstyles">
        <property name="toolTip">
         <string>Load Styles</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../tikzit.qrc">
          <normaloff>:/images/document-open.svg</normaloff>:/images/document-open.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonEditTikzstyles">
        <property name="toolTip">
         <string>Edit styles</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../tikzit.qrc">
          <normaloff>:/images/text-x-generic_with_pencil.svg</normaloff>:/images/text-x-generic_with_pencil.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonRefreshTikzstyles">
        <property name="toolTip">
         <string>Refresh styles</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../tikzit.qrc">
          <normaloff>:/images/refresh.svg</normaloff>:/images/refresh.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="currentCategory"/>
    </item>
    <item>
     <widget class="QListView" name="styleListView">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="edgeStyleListView">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../tikzit.qrc"/>
 </resources>
 <connections/>
</ui>
